dashboard/app/reporting_email.go

```diff
@@ -392,7 +392,7 @@ func handleTestCommand(c context.Context, info *bugInfoResult, msg *email.Email)
 	err := handleTestRequest(c, &testReqArgs{
```

Comment:
Add a check somewhere here that email test request related to AccessPublic bugs and a test where email request tries to submit testing request for external reporting bug/non-public bug.